specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -48,13 +47,13 @@ a middle ground between these two sets of requirements.
 
 In terms of completeness, the Web Profile
 offers a complete stack, with technologies addressing presentation and
-state management (JavaServer Faces, JavaServer Pages), core web
-container funtionality (Servlet), business logic (Enterprise JavaBeans
-Lite), transactions (Java Transaction API), persistence (Java
+state management (JakartaServer Faces, JakartaServer Pages), core web
+container funtionality (Servlet), business logic (Enterprise JakartaBeans
```

Comment:
Servlet -> Jakarta Servlet (to differentiate from "Oracle" Servlet). Jakarta Enterprise Beans.